opsdroid/connector/slack/events.py

```diff
@@ -42,3 +43,57 @@ def __init__(self, blocks, *args, **kwargs):
         self.blocks = blocks
         if isinstance(self.blocks, list):
             self.blocks = json.dumps(self.blocks)
+
+
+class InteractiveAction(Event):
+    """Super class to represent Slack interactive actions."""
+
+    def __init__(self, payload, *args, **kwargs):
+        """Create object with minimum properties."""
+        super().__init__(*args, **kwargs)
+        self.payload = payload
+
+    async def respond(self, message):
```

Comment:
Users should be able to respond with any event type and if it is not supported it should handle that with a warning in the log.

You could check if `message` is a string here and if then call `super().respond`.

Reply:
Done.